packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -205,56 +164,68 @@ export class MarketOperationUtils {
             : Promise.resolve([]);
 
         const [
-            [tokenDecimals, orderFillableAmounts, ethToMakerAssetRate, ethToTakerAssetRate, dexQuotes, rawTwoHopQuotes],
-            rfqtIndicativeQuotes,
```

Comment:
Are indicative quotes always fetched after sampling/optimization now?

Reply:
Yeah so the RFQ maker can get the price to beat. Same flow for firm/indicative. Discover price to beat, RFQ, re-optimize